winui/Shimmer/Customization.md

```diff
@@ -0,0 +1,119 @@
+---
+layout: post
+title: Customization in WinUI Shimmer control | Syncfusion
+description: Learn how to customize the appearance of the Syncfusion WinUI Shimmer(SfShimmer) control using various properties like Fill, Wave Color, Wave Width, Repeat Count and Wave Duration.
+platform: WinUI
+control: Shimmer
+documentation: ug
+---
+
+# Customization in WinUI Shimmer
+
+This section explains how to customize the appearance of the Syncfusion WinUI Shimmer (SfShimmer) control using various properties. You can modify the following aspects of the Shimmer effect:
+
+## Fill
+
+The `Fill` property allows you to set the background color of the Shimmer view.
+
```

Comment:
@GilbertmanojSF3971 The Fill property allows you to specify the background color of the shimmer view.

Reply:
@Manivannan-E Sir changed the content to The Fill property allows you to specify the background color of the shimmer view.